skeleton/docker-entrypoint.sh

```diff
@@ -163,6 +163,14 @@ elif  [[ "$1" == "run" ]]; then
   exec $sudo $VENVBIN/zconsole run etc/${CONF} "${@:2}"
 elif  [[ "$1" == "addzopeuser" ]]; then
   exec $sudo $VENVBIN/addzopeuser -c etc/${CONF} "${@:2}"
+elif  [[ "$1" == "pack" ]]; then
```

Comment:
I can't tell what command provides the parameter `$1`. Is it `make` or something else? In any case, docs please!

Reply:
@stevepiercy This is a bash script, and it's just plain Bash. The first argument to the command is available as `$1`. So this runs if the user called this script using `./docker-entrypoint.sh pack`

Reply:
@davisagli gotcha. So `cd skeleton && ./docker-entrypoint.sh pack` should do the job? Let's get some docs of its complete usage. @erral has an open PR to put it as a [recipe](https://github.com/plone/documentation/pull/1886/files), but I think it should go in the existing [Advanced usage in plone/plone-backend](https://6.docs.plone.org/install/containers/images/backend.html#advanced-usage).

Reply:
@stevepiercy No, there is no situation where that's what you would do. The skeleton folder is here in the repository and is the source for files that go in the image, but there is no skeleton folder in the image itself.

It will be easiest to run this command in an already running plone-backend container with `docker exec -it [container id] ./docker-entrypoint.sh pack`.

In a Docker Swarm configuration it's also possible to set up a separate service which is triggered periodically using https://github.com/crazy-max/swarm-cronjob

The whole containers section needs a complete reorganization, so I'm not going to get hung up on where it goes for now.